### DISCUSSION TITLE
Add an option to "open new" instead of native "move to" behavior when select an app?

### DISCUSSION THREAD

**Discussion by lapinvert** [rxhanson/Charmstone-Community#18] [Ideas]
Hi!

Let me explain my use case:
I work with multiple screens and spaces. 
I have a Charmstone shortcut to the `Downloads` folder.
If there's already a `Downloads` window opened _somewhere_ in my many screens and spaces, and then I trigger it with Charmstone from somewhere else, it will move my current space / screen / focus, to where that window is, when really, I just wanted to quickly open it from _where I was_.
I understand this must be some native OS behavior but I would guess Charmstone _could_ force re-open a new window, so having the ability to set that option on a per-shortcut basis would be awesome. A checkbox like this for example:

- [ ] "open new "

Or, it could also be a modifier key to trigger this behavior when selecting the shortcut.

**Comment by rxhanson** [comment]
Thanks for the feedback. A pro and con of Charmstone is that it is sandboxed and available on the App Store. This means that it cannot tell an application to open a new window, but this is possible by creating a script to do it, and adding the script to Charmstone. This is a lot more hands-on, but also a lot more powerful if you are willing to spend the time to do it.

Here is an example script, made for Terminal: https://github.com/rxhanson/Charmstone-Community/blob/main/Scripts/NewTerminalWindow.scpt

You'll want to edit your script in Script Editor (already on your Mac), and export it as an Application to assign in Charmstone. You should also be able to update the icon if desired using the steps here: https://9to5mac.com/2021/11/08/change-mac-icons/
